src/components/Tabs.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Tabs.scss';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (id: string) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const content = tabs.find(tab => tab.id === selectedTabId)?.content;
+
+  return (
+    <div className="Tabs">
+      <ul className="Tabs__list">
+        {tabs.map(tab => (
+          <li key={tab.id} className="Tabs__item">
+            <button
+              type="button"
+              className={`Tabs__button ${tab.id === selectedTabId && 'Tabs__button--active'}`}
+              onClick={() => onTabSelected(tab.id)}
```

Comment:
![image](https://user-images.githubusercontent.com/37161315/146893636-6adeff3a-5765-4ec6-abac-f10da15eadb5.png)

Reply:
Don't call the callback if the tab wasn't changed